### DISCUSSION TITLE
Default resolution picked by player always way too low

### DISCUSSION THREAD

**Discussion by ghost** [ViewTube/viewtube#2800] [Q&A] [answered]
Hello there. Viewtube is almost perfect. However, there's something wrong, as it always picks incorrect [too low, practically always the lowest available] video resolution. I have quite good internet connection and other players have no issues picking proper [HD usually] resolution, so I am convinced it's related specifically to Viewtube. Is this known issue? I cannot imagine issue like this being persistent across multiple releases. It's super inconvenient.

Did anyone have issue like this? Initially I thought it might be due to some plugins I use within the browser, but I tried without them and the problem persists.

I'd be grateful for any suggestions, tips, explanation. 

BTW such a great project! You beat Invidious instantly, so it's great there's good competition.

**Comment by moisout** [comment]
This issue should be fixed with the new video player, which is releasing very soon!

**Reply by ghost** [reply, marked as answer]
just testing dev version from docker hub and indeed it's working great! picks up HD quickly. Great work (y)
